bibliopixel/animation/animation.py

```diff
@@ -15,6 +15,8 @@ class STATE(IntEnum):
 
 class BaseAnimation(object):
     free_run = False
+    name = ''
+    parent_data = None
```

Comment:
NO NO NO. Not going to call it parent data. That's to damn confusing to anyone but you and I. The remote configs will look weird... because parent makes no sense out of the context of our own understanding of the code class relationships. Users have to use this to. No.